documentation/src/main/asciidoc/topics/proc_cli_getting_server_reports.adoc

```diff
@@ -0,0 +1,26 @@
+[id='server_reports-{context}']
+= Getting Diagnostic Reports for {brandname} Servers
+{brandname} servers provide aggregated reports in `tar.gz` archives that
+contain diagnostic information about both the {brandname} server and the host.
+The report provides details about CPU, memory, open files, network sockets and
+routing, threads, in addition to configuration and log files.
+
+.Prerequisites
+
+* Open a CLI connection to {brandname}.
+
+.Procedure
+
+. Use the `server report` command to download a `tar.gz` archive:
++
+----
+[//containers/default]> server report
+Downloaded report '${brandname}-<hostname>-<timestamp>-report.tar.gz'
```

Comment:
@oraNod is there a way to resolve attributes inside code blocks as above?
This is not being resolved. The HTML shows `${brandname}`

Reply:
Actually we don't want ${brandname} here, because that is the capitalized version